docs/components/bucket.md

```diff
@@ -101,23 +103,23 @@ Flags:
       --version                Show application version.
       --log.level=info         Log filtering level.
       --log.format=logfmt      Log format to use.
-      --tracing.config-file=<tracing.config-yaml-path>
+      --tracing.config-file=<file-path>
                                Path to YAML file that contains tracing
                                configuration. See format details:
                                https://thanos.io/tracing.md/#configuration
-      --tracing.config=<tracing.config-yaml>
+      --tracing.config=<content>
                                Alternative to 'tracing.config-file' flag.
-                               Tracing configuration in YAML. See format
-                               details:
+                               Content of YAML file that contains tracing
```

Comment:
nit: I understand that it's composed of the generic prefix and flag help but the resulting wording "content of file that contains" sounds bit weird to me.  "Content of YAML file with tracing configuration." maybe? 
But perhaps it is just me.

Reply:
happy to change